doc/nrf/app_dev/config_and_build/multi_image.rst

```diff
@@ -339,11 +339,6 @@ For example, in the |NCS| applications and samples that use different :ref:`buil
 
 The listing below describes how to leverage this functionality, where ``ACI_NAME`` is the name of the child image to which the configuration will be applied.
 
-.. literalinclude:: ../../../../cmake/multi_image.cmake
-    :language: c
-    :start-at: It is possible for a sample to use a custom set of Kconfig fragments for a
-    :end-before: set(ACI_CONF_DIR ${config_dir}/child_image)
-
 When you are using :ref:`app_build_additions_build_types` and the configuration name has been inferred, the child image Kconfig overlay file is searched at :file:`child_image/<ACI_NAME>_<name>.conf`.
```

Comment:
Please update the entire page or remove it, given the note at the top of the page.

Other pages that need to be updated:
- config_and_build_system.rst - sections "Child images" and "Multi-image builds"
- multi_image_nrf5340.rst - entire page needs to be reviewed or removed